internal/utils.go

```diff
@@ -15,3 +15,10 @@ var uriReg = regexp.MustCompile(`(^https?:\/\/(www\.))?[-a-zA-Z0-9@:%._\+~#=]{2,
 func IsUri(uri string) bool {
 	return uriReg.MatchString(uri)
 }
+
+func PickArgs(args []string, ptr int) string {
+	if len(args) >= ptr {
+		return args[ptr]
+	}
+	panic("Invalid prt")
```

Comment:
I don't think this is a fatal error, maybe we should  print error message in console and exit process.

Reply:
Or we could provide `logger.fatal()` method to log err then exit.

Reply:
Out of bounds is only runtime behavior. So have some ideas?